service/src/test/java/uk/nhs/adaptors/gp2gp/common/service/XPathServiceTest.java

```diff
@@ -41,4 +43,16 @@ public void When_GetNodeValueWithInvalidXPath_Expect_ThrowsRuntimeException() {
             .withMessageContaining("!!!NOT XPATH");
     }
 
+    @Test
+    @SneakyThrows
+    public void When_GetNodesWithValidXPath_Expect_NodeListReturned() {
+        var document = new XPathService().parseDocumentFromXml("<elements><element>value</element><element>value2</element></elements>");
+        NodeList nodes = new XPathService().getNodes(document, "//element");
+        assertThat(nodes.getLength()).isEqualTo(2);
```

Comment:
Is this needed? It's also in the `assertAll` block

Reply:
removed